Dockerfile

```diff
@@ -10,11 +10,14 @@ ENV SECRET_KEY_BASE $secret_key_base
 
 RUN apt-get update
 
+ENV LANG C.UTF-8
 RUN curl -sL https://deb.nodesource.com/setup_5.x | bash && \
-    apt-get install -y nodejs
+    apt-get install -y nodejs locales
 
 ADD Gemfile /tmp/Gemfile
 ADD Gemfile.lock /tmp/Gemfile.lock
+RUN mkdir -p /tmp/engines
```

Comment:
This is not necessary because the `ADD` command is already copying all the folders.